app/src/androidTest/java/org/simple/clinic/storage/migrations/Migration36AndroidTest.kt

```diff
@@ -0,0 +1,284 @@
+package org.simple.clinic.storage.migrations
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.simple.clinic.string
+
+class Migration36AndroidTest : BaseDatabaseMigrationTest(
+    fromVersion = 35,
+    toVersion = 36
+) {
+
+  @Test
+  fun migrate_prescription_from_35_to_36() {
+    val tableName = "PrescribedDrug"
+    val uuid = "drug-uuid"
+    before.execSQL("""
+      INSERT INTO $tableName VALUES(
+      '$uuid',
+      'drug',
+      'dosage',
+      'rxNormCode',
+      0,
+      1,
+      'patientUuid',
+      'facilityUuid',
+      'PENDING',
+      'created-at',
+      'updatedAt',
+      'null',
+      'recorded-at')
+    """)
+
+    after.query("SELECT * FROM $tableName").use {
+      assertThat(it.count).isEqualTo(1)
+      assertThat(it.getColumnIndex("recordedAt")).isEqualTo(-1)
+      it.moveToNext()
+      assertThat(it.columnCount).isEqualTo(12)
+      assertThat(it.string("uuid")).isEqualTo(uuid)
+    }
+  }
+
+  @Test
+  fun migrate_appointment_from_35_to_36() {
+    val tableName = "Appointment"
+    val appointmentUuid = "uuid"
+
+    before.execSQL("""
+      INSERT INTO $tableName VALUES(
+      '$appointmentUuid',
+      'patientUuid',
+      'facility-uuid',
+      'scheduled-date',
+      'status',
+      'cancel-reason',
+      'remind-on',
+      '1',
+      'manual',
+      'PENDING',
+      'created-at',
+      'updated-at',
+      'null',
+      'recordedAt')
+    """)
+
+    after.query("SELECT * FROM $tableName").use {
+      assertThat(it.count).isEqualTo(1)
+      assertThat(it.getColumnIndex("recordedAt")).isEqualTo(-1)
+      it.moveToNext()
+      assertThat(it.columnCount).isEqualTo(13)
+      assertThat(it.string("uuid")).isEqualTo(appointmentUuid)
+    }
+  }
+
+  @Test
+  fun migrate_medical_histories_from_35_to_36() {
+    val tableName = "MedicalHistory"
+    val mhUuid = "uuid"
+
+    before.execSQL("""
+      INSERT INTO $tableName VALUES(
+      '$mhUuid',
+      'patientUuid',
+      0,
+      1,
+      0,
+      1,
+      0,
+      1,
+      'PENDING',
+      'created-at',
+      'updated-at',
+      'null',
+      'recorded-at')
+    """)
+
+    after.query("SELECT * FROM $tableName").use {
+      assertThat(it.count).isEqualTo(1)
+      assertThat(it.getColumnIndex("recordedAt")).isEqualTo(-1)
+      it.moveToNext()
+      assertThat(it.columnCount).isEqualTo(12)
+      assertThat(it.string("uuid")).isEqualTo(mhUuid)
```

Comment:
Can you explain what is happening here?

Reply:
This is mostly a copy from an older migration. In a previous migration, we had added the `recordedAt` column to the medical history table, but we decided not to use that field so we dropped it in this migration.